src/class-metadata.php

```diff
@@ -603,7 +603,7 @@ private function get_bottom_level_term( int $post_id, string $taxonomy_name ): s
 		$terms_not_parents = array_intersect_key( $terms, $term_ids_not_parents );
 		// remove array index keys.
 		$terms_not_parents_cleaned = array();
-		foreach ( $terms_not_parents as $index => $value ) {
+		foreach ( $terms_not_parents as $value ) {
 			$terms_not_parents_cleaned[] = $value;
 		}
```

Comment:
If you're trying to simplify it from an associative array to an indexed array, then consider using [`array_values()` function](https://www.php.net/manual/en/function.array-values.php) instead of creating a foreach loop.

Reply:
Otherwise, this looks like it's looping through one array and assigning them as terms to another, which seems redundant.